src/lizaalert/homeworks/admin.py

```diff
@@ -1,25 +1,63 @@
+from django import forms
 from django.contrib import admin
+from rest_framework.decorators import permission_classes
+from tinymce.widgets import TinyMCE
 
 from lizaalert.courses.models import Lesson
 from lizaalert.homeworks.models import Homework
+from lizaalert.homeworks.permissions import IsReviewerOrSuperUser
+
+
+class HomeworkAdminForm(forms.ModelForm):
+    class Meta:
+        model = Homework
+        fields = ("reviewer", "status", "lesson", "text", "subscription", "required")
+        widgets = {
+            "text": TinyMCE(attrs={"cols": 80, "rows": 30}),
```

Comment:
высота с 30 символов - это много, давай оставим  10

```suggestion
            "text": TinyMCE(attrs={"cols": 80, "rows": 10}),
```